green-recorder

```diff
@@ -37,7 +37,11 @@ else:
   config.add_section('Options')
   config.set('Options', 'frames', '30')
   config.set('Options', 'delay', '0')
-  config.set('Options', 'folder', "file://" + GLib.get_user_special_dir(GLib.USER_DIRECTORY_VIDEOS))
+  config.set('Options', 'folder',"file://" + os.path.expanduser('~')) # set default folder to home
+  try:
+      config.set('Options', 'folder', "file://" + GLib.get_user_special_dir(GLib.USER_DIRECTORY_VIDEOS))#set default folder to video if exits
```

Comment:
You could also check whether `GLib.get_user_special_dir()` returns `None` (which it does, if a requested folder is not available). Would avoid handling an arbitrary exception for this.